src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbOperations.java

```diff
@@ -45,5 +46,7 @@ DocumentCollection createCollectionIfNotExists(DocumentDbEntityInformation infor
 
     <T> List<T> find(DocumentQuery query, Class<T> entityClass, String collectionName);
 
+    <T> List<T> findAll(Sort sort, Class<T> domainClass, String collectionName);
```

Comment:
How about other find operation with Sort? i.e., find with some conditions and sort result. Probably something like `find(DocumentQuery query, Sort sort,  Class<T> entityClass, String collectionName)`